src/AddTodoForm.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+  const AddTodoForm = () => {
+    return (
+        <form>
+            <label htmlFor="todoTitle"> Title </label>
+            <input id="todoTitle"></input>
```

Comment:
What kind of input is this? Is there a property that could clearly define it?

Reply:
done:)